.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -5,31 +5,32 @@ about: Report a bug with this project. Make sure you've read "Things To Know Bef
 
 ---
 
-### Things That I, The Bug Reporter, Have Done Before Submitting
+_Fill out the following document, replacing parts with _italic emphasis_ with your issue information_
+
+### Before Submitting This Issue:
 - [ ] I have ensured that this is a bug. Where I was not sure, I checked the [forum](https://forum.pi-top.com/c/pi-top-software), and opened a discussion there.
 - [ ] I have read [Things To Know Before Submitting A Bug Report](https://github.com/pi-top/pi-top-Python-SDK/wiki/Things-To-Know-Before-Submitting-A-Bug-Report) before filling this out.
 
 ### Summary
-Concise summary of when and how the bug occurred, using code examples where relevant. Include links to related forum posts, if relevant.
+_Concise summary of when and how the bug occurred, using code examples and output logs where relevant. Include links to related forum posts, if relevant._
 
-### Screenshot/Video Capture
-This may not be relevant, but should be provided where possible. Read [How To Get System Information](https://github.com/pi-top/pi-top-Python-SDK/wiki/How-To-Get-System-Information) for assistance with this.
+### Screenshots / Photos / Link to Video
+_Additional visual information may help to explain how the bug was produced and/or how it manifests._
 
 ### Expected vs. Actual Results
-What were you expecting to have happen?
+_What were you expecting to have happen?_
 
 ### Steps To Reproduce
-Is it possible to reproduce the bug repeatably? What steps might someone trying to help need to do to see what you are seeing?
+_Is it possible to reproduce the bug repeatably? What steps might someone trying to help need to do to see what you are seeing?_
+
+### Environment Information
+_This is valuable to identify the root of the problem. Read [How To Get System Information](https://github.com/pi-top/pi-top-Python-SDK/wiki/How-To-Get-System-Information) for assistance with this._
 
-### Environment
-Read [How To Get System Information](https://github.com/pi-top/pi-top-Python-SDK/wiki/How-To-Get-System-Information) for assistance with this.
+_Drag and drop pt-diagnostics.zip file here and/or provide:_
```

Comment:
I don't think we should necessarily encourage this, as it will likely have too much information. In particular, it could have personal information. It was never really designed for this use-case

Reply:
Ok, I have removed this and _somewhat_ improved the [system information](https://github.com/pi-top/pi-top-Python-SDK/wiki/How-To-Get-System-Information) wiki page